widgets/modelviewer/widget.maifest.config.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  defineWidgetManifest,
+  mergeWidgetManifests,
+} from '@equinor/fusion-framework-cli';
+
+export default defineWidgetManifest((env, { base }) => {
```

Comment:
Is this file used runtime in the browser? Otherwise we can get version, endpoint etc from the package json. To relieve ourself of the pain of maintaining both.

Reply:
Fixed so that the app fetches from package.json